content/intro-to-storybook/react-native/en/simple-component.md

```diff
@@ -21,221 +21,168 @@ This process is similar to [Test-driven development](https://en.wikipedia.org/wi
 
 ## Get set up
 
-First, let’s create the task component and its accompanying story file: `components/Task.js` and `components/Task.stories.js`.
+Let's open `.storybook/main.js` and take a look
+
+```js:title=.storybook/main.js
+module.exports = {
+  stories: ["../components/**/*.stories.?(ts|tsx|js|jsx)"],
+  addons: [
+    "@storybook/addon-ondevice-controls",
+    "@storybook/addon-ondevice-actions",
+  ],
+};
+```
+
+If you check the `stories` property you'll see that Storybook is looking for stories in the `components` folder.
+
+In React Native Storybook we use the config in `main.js` to generate a file called `storybook.requires.js`, this is because React Native doesn't support dynamic imports yet. This file is used to load all the stories in the project and it also imports your addons.
+
+This file gets generated when you run `yarn storybook` to start storybook or `yarn storybook-generate` if you just want to regenerate the `storybook.requires.js` file. Whenever you find that a story is not being loaded, you can try regenerating this file and it should show up.
```

Comment:
@dannyhw, can you combine this into a single paragraph that highlights the behavior and the caveat?